tests/ftests/054-sudo-g_flag_controller_only_systemd-v1.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Advanced cgget functionality test - '-b' '-g' <controller> (cgroup v1)
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from cgroup import Cgroup, CgroupVersion
+from run import Run, RunError
+import consts
+import ftests
+import sys
+import os
+
+CONTROLLER = 'cpu'
+SYSTEMD_CGNAME = 'cg_in_scope'
+OTHER_CGNAME = 'cg_not_in_scope'
+TMP_CGNAME = 'tmp_cg'
+
+SLICE = 'libcgroup'
+SCOPE = 'test054'
+
+CONFIG_FILE = ''
+CONFIG_FILE_NAME = os.path.join(os.getcwd(), '051cgconfig.conf')
+
+DEFAULT_SYSTEMD_CGROUP_NAME = '/run/libcgroup/systemd'
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if CgroupVersion.get_version('cpu') != CgroupVersion.CGROUP_V1:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires the cgroup v1 cpu controller'
+        return result, cause
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+
+    return result, cause
+
+
+def setup(config):
+    # Let's remove the existing default systemd cgroup file
+    # so that the temp cgroup we create is at the know path.
+    if os.path.exists(DEFAULT_SYSTEMD_CGROUP_NAME):
+        os.remove(DEFAULT_SYSTEMD_CGROUP_NAME)
+
+    # The job of the tmp cgroup is to create a task, that will be
+    # used by the systemd as pid for the scope.
+    Cgroup.create(config, CONTROLLER, TMP_CGNAME)
+    config.process.create_process_in_cgroup(config, CONTROLLER, TMP_CGNAME)
```

Comment:
Why are you creating a process in a cgroup only to have it be moved to the newly-created scope?  Wouldn't calling `config.create_process()` be simpler?

Reply:
yeah, it will be way more elegant.